src/featurization_utils/pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "featurization_utils_module"
+version = "0.0.0"
+description = "Utility package for GFF 3D organoid profiling pipeline"
+requires-python = ">=3.11"
+authors = [ { name = "Your Name"} ]
+license = { text = "MIT" }
+
+dependencies = [
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest",
+    "black",
+    "ruff",
+    "pre-commit",
+    "ipykernel",
+]
+
+[project.scripts]
+area_size_shape_utils_gpu = "area_size_shape_utils_gpu:main"
+area_size_shape_utils = "area_size_shape_utils:main"
+colocalization_utils_gpu = "colocalization_utils_gpu:main"
+colocalization_utils = "colocalization_utils:main"
+granularity_utils = "granularity_utils:main"
+intensity_utils = "intensity_utils:main"
+neighbors_utils = "neighbors_utils:main"
+resource_profiling_util = "resource_profiling_util:main"
+texture_utils = "texture_utils:main"
+loading_classes = "loading_classes:main"
```

Comment:
How do these get invoked? I might find this later in the review, adding a comment just in case. If you don't need to invoke these by script, consider removing this block.